src/headerData.json

```diff
@@ -0,0 +1,9 @@
+{
+  "nav": [
+    { "title":"Campañas","url":"https://app2.fromdoppler.com/Campaigns/Draft/","isEnabled":false,"isSelected":false,"subNav":[{"title":"Borradores","url":"https://app2.fromdoppler.com/Campaigns/Draft/","isEnabled":false,"isSelected":false,"idHTML":"campaignDraftMenu","$$hashKey":"object:32"},{"title":"Programadas","url":"https://app2.fromdoppler.com/Campaigns/Scheduled/","isEnabled":false,"isSelected":false,"idHTML":"campaignScheduledMenu","$$hashKey":"object:33"},{"title":"Enviadas","url":"https://app2.fromdoppler.com/Campaigns/Sent/","isEnabled":false,"isSelected":false,"idHTML":"campaignSentMenu","$$hashKey":"object:34"},{"title":"Test A/B","url":"https://app2.fromdoppler.com/Campaigns/TestAB/","isEnabled":false,"isSelected":false,"idHTML":"campaignTestABMenu","$$hashKey":"object:35"}],"idHTML":"campaignMenu","$$hashKey":"object:6"},
+    {"title":"Listas","url":"https://app2.fromdoppler.com/Lists/SubscribersList","isEnabled":false,"isSelected":false,"subNav":[{"title":"Listas Principales","url":"https://app2.fromdoppler.com/Lists/SubscribersList/","isEnabled":false,"isSelected":false,"idHTML":"listMainMenu","$$hashKey":"object:41"},{"title":"Segmentos","url":"https://app2.fromdoppler.com/Lists/Segment/","isEnabled":false,"isSelected":false,"idHTML":"listSegmentMenu","$$hashKey":"object:42"},{"title":"Suscriptores","url":"https://app2.fromdoppler.com/Lists/MasterSubscriber/","isEnabled":false,"isSelected":false,"idHTML":"listMasterSubscriberMenu","$$hashKey":"object:43"},{"title":"Campos Personalizados","url":"https://app2.fromdoppler.com/Lists/MasterCustomFields/","isEnabled":false,"isSelected":false,"idHTML":"listCustomFieldMenu","$$hashKey":"object:44"},{"title":"Formularios","url":"https://app2.fromdoppler.com/Lists/Form/","isEnabled":false,"isSelected":false,"idHTML":"listFormMenu","$$hashKey":"object:45"}],"idHTML":"listMenu","$$hashKey":"object:7"},
+    {"title":"Automation","url":"https://app2.fromdoppler.com/Automation/Automation/AutomationApp/","isEnabled":false,"isSelected":false,"subNav":[],"idHTML":"automationMenu","$$hashKey":"object:8"},
+    {"title":"Reportes","url":"https://app2.fromdoppler.com/Campaigns/Reports/","isEnabled":false,"isSelected":false,"subNav":[],"idHTML":"reportMenu","$$hashKey":"object:9"},
+    {"title":"Plantillas","url":"https://app2.fromdoppler.com/Templates/Main/","isEnabled":false,"isSelected":true,"idHTML":"templateMenu","$$hashKey":"object:10"}
+  ]
+}
```

Comment:
What do you think about prettifying this json?

Reply:
it's posible?

Reply:
Sure! I just did it using [this tool](https://jsonformatter.curiousconcept.com/), there are other tools, or you also could do it manually.

```json
{
  "nav":[
    {
      "title":"Campañas",
      "url":"https://app2.fromdoppler.com/Campaigns/Draft/",
      "isEnabled":false,
      "isSelected":false,
      "subNav":[
        {
          "title":"Borradores",
          "url":"https://app2.fromdoppler.com/Campaigns/Draft/",
          "isEnabled":false,
          "isSelected":false,
          "idHTML":"campaignDraftMenu",
          "$$hashKey":"object:32"
        },
        {
          "title":"Programadas",
          "url":"https://app2.fromdoppler.com/Campaigns/Scheduled/",
          "isEnabled":false,
          "isSelected":false,
          "idHTML":"campaignScheduledMenu",
          "$$hashKey":"object:33"
        },
        {
          "title":"Enviadas",
          "url":"https://app2.fromdoppler.com/Campaigns/Sent/",
          "isEnabled":false,
          "isSelected":false,
          "idHTML":"campaignSentMenu",
          "$$hashKey":"object:34"
        },
        {
          "title":"Test A/B",
          "url":"https://app2.fromdoppler.com/Campaigns/TestAB/",
          "isEnabled":false,
          "isSelected":false,
          "idHTML":"campaignTestABMenu",
          "$$hashKey":"object:35"
        }
      ],
      "idHTML":"campaignMenu",
      "$$hashKey":"object:6"
    },
    {
      "title":"Listas",
      "url":"https://app2.fromdoppler.com/Lists/SubscribersList",
      "isEnabled":false,
      "isSelected":false,
      "subNav":[
        {
          "title":"Listas Principales",
          "url":"https://app2.fromdoppler.com/Lists/SubscribersList/",
          "isEnabled":false,
          "isSelected":false,
          "idHTML":"listMainMenu",
          "$$hashKey":"object:41"
        },
        {
          "title":"Segmentos",
          "url":"https://app2.fromdoppler.com/Lists/Segment/",
          "isEnabled":false,
          "isSelected":false,
          "idHTML":"listSegmentMenu",
          "$$hashKey":"object:42"
        },
        {
          "title":"Suscriptores",
          "url":"https://app2.fromdoppler.com/Lists/MasterSubscriber/",
          "isEnabled":false,
          "isSelected":false,
          "idHTML":"listMasterSubscriberMenu",
          "$$hashKey":"object:43"
        },
        {
          "title":"Campos Personalizados",
          "url":"https://app2.fromdoppler.com/Lists/MasterCustomFields/",
          "isEnabled":false,
          "isSelected":false,
          "idHTML":"listCustomFieldMenu",
          "$$hashKey":"object:44"
        },
        {
          "title":"Formularios",
          "url":"https://app2.fromdoppler.com/Lists/Form/",
          "isEnabled":false,
          "isSelected":false,
          "idHTML":"listFormMenu",
          "$$hashKey":"object:45"
        }
      ],
      "idHTML":"listMenu",
      "$$hashKey":"object:7"
    },
    {
      "title":"Automation",
      "url":"https://app2.fromdoppler.com/Automation/Automation/AutomationApp/",
      "isEnabled":false,
      "isSelected":false,
      "subNav":[

      ],
      "idHTML":"automationMenu",
      "$$hashKey":"object:8"
    },
    {
      "title":"Reportes",
      "url":"https://app2.fromdoppler.com/Campaigns/Reports/",
      "isEnabled":false,
      "isSelected":false,
      "subNav":[

      ],
      "idHTML":"reportMenu",
      "$$hashKey":"object:9"
    },
    {
      "title":"Plantillas",
      "url":"https://app2.fromdoppler.com/Templates/Main/",
      "isEnabled":false,
      "isSelected":true,
      "idHTML":"templateMenu",
      "$$hashKey":"object:10"
    }
  ]
}
```

Reply:
Does it make sense?